utilities/cli/holohub.py

```
@@ -2053,6 +2053,37 @@ def _add_to_cmakelists(self, project_name: str) -> None:
             print(Color.red(f"Failed to add application to applications/CMakeLists.txt: {str(e)}"))
             print(Color.red("Please add the application manually to applications/CMakeLists.txt"))
 
+    def validate_generated_metadata(self, metadata_path: Path, schema_root: Optional[Path]) -> None:
+        """Validate metadata.json for the newly created project."""
+        import json
+
+        import utilities.metadata.metadata_validator as metadata_validator
+
+        if not schema_root:
+            # No schema installed – skip validation.
```

Ruff (0.14.6)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)